docs/tls.md

```diff
@@ -0,0 +1,28 @@
+# Aerospike JDBC TLS Configuration Parameters
+
+The Aerospike JDBC driver can be configured to use SSL through the JDBC connection URL. 
+
+Below are the tls-related configs for aerospike-jdbc (example with value):
+```tlsEnabled true
```

Comment:
New line after ```

Reply:
have added newline before and after the code block